dist/formats/organisation/frontend/schema.json

```diff
@@ -334,6 +334,13 @@
             "null"
           ]
         },
+        "alternative_format_contact_email": {
+          "description": "The organisation's email for requested an alternative format for attachments",
```

Comment:
I think you mean 'requesting' here instead of 'requested'.

Reply:
Ah, thanks for checking - fixed :-).